README.md

```diff
@@ -27,7 +27,12 @@ commits and tags:
 * works without any VCS, but happily reads tag information from and writes
   commits and tags to Git and Mercurial if available
 * just handles text files, so it's not specific to any programming language
-* supports Python2, Python3 and Pypy
+* supports Python 3 and PyPy3
+
+If you want to use Python 2, use `pip>=9` and you'll get the last supported version,
+or pin `bump2version<1`.
```

Comment:
I'd phrase it slightly different:
```suggestion
When using Python 2, pip 9.0.0 and higher will install a compatible version using [PEP440's](https://www.python.org/dev/peps/pep-0440/) [python_requires](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires). With older pip versions a manual pin to `bump2version < 1` is needed.
```

Reply:
Looks good, please could you add a space to "PEP 440" in the suggestions? That's how they're written:

https://www.python.org/dev/peps/pep-0440/

Plus add code formatting backticks around `python_requires`?